src/RNVisaCheckout.js

```diff
@@ -1,16 +1,26 @@
-
+// @flow
 import { NativeModules } from 'react-native';
 
 const { RNVisaCheckout } = NativeModules;
 
-export const configureProfileAsync = async (environment, apiKey, profileName) => {
-  const data = await RNVisaCheckout.configureProfile(environment, apiKey, profileName);
-  return data;
+type ConfigureResponse = {
+  status: number,
+};
+export const configureProfileAsync = async (environment: number, apiKey: string, profileName: string): Promise<ConfigureResponse> => {
+  return await RNVisaCheckout.configureProfile(environment, apiKey, profileName);
 };
 
-export const checkoutAsync = async (total, currency) => {
-  const data = await RNVisaCheckout.checkout(total, currency);
-  return data;
+type CheckoutResponse = {
+  callId: string,
+  lastFourDigits: string,
+  country: number,
+  postalCode: string,
+  paymentMethodType: string,
+  cardBrand: number,
+  encryptedKey: string,
+}
```

Comment:
There is a problem with this. In iOS, country and cardBrand are numbers, in Android, they are strings. Change country and cardboard to string || number?

Reply:
Or change in native code to always return number, just would need a helper function to map them to their correct values

Reply:
last.

If we can keep consistency between both system it's the best option.

Reply:
Ok. Will do.